lstchain/calib/camera/calib.py

```diff
@@ -78,7 +80,7 @@ def gain_selection(waveform, charges, pulse_time, threshold):
     return combined_image, combined_pulse_time
 
 
-
+@deprecated('28/06/2019', message='channel selection is now performed at <= R1 calibration level')
```

Comment:
not sure if we want to leave this here or get rid of it to clean up

Reply:
I wanted to leave it until a release that might happen with ctapipe next release (hopefully) and for lst custom calibration